Rogerjw/Broadcaster
# ch(endpoints):Following conventions

#### What does this PR do?
- remove all the user validating codes from controllers
- add some validating codes in the auth.middleware
-delete to some folders which should be included in .gitignore
#### Description of Task to be completed?
- writing clean codes that follow conventions
#### How should this be manually tested?
- Clone this repository anywhere on your computer.
- use Postman, to test see if you didn't break anything
#### What are the relevant pivotal tracker stories?
[#169950991](https://www.pivotaltracker.com/story/show/169950991)
#### Screenshots (if appropriate)
no validation in the controllers
![image](https://user-images.githubusercontent.com/43696555/69623366-6caba000-104b-11ea-9e20-a8fdff60b8e9.png)
